src/js/components/devices/authorized-devices.js

```diff
@@ -78,15 +78,10 @@ export const Authorized = props => {
   const handleResize = () => setTimeout(() => setSize({ height: window.innerHeight, width: window.innerWidth }), 500);
 
   useEffect(() => {
-    onSelectionChange([]);
     clearAllRetryTimers(setSnackbar);
     if (!filters.length && selectedGroup && groupFilters.length) {
       setDeviceFilters(groupFilters);
     }
-    clearInterval(timer);
-    // no group, no filters, all devices
-    timer = setInterval(getDevices, refreshDeviceLength);
-    getDevices();
```

Comment:
so now it will only get once when mounted, right?

Reply:
well, both effects/ trigger were run when mounted and the second trigger also is run on any relevant state change afterwards... This first one was basically a duplicate (based on the previous implementation, where these effects were split up) + relying only on the second one allows triggering it from a parent component...